bundle-workflow/tests/tests_manifests/test_test_manifest.py

```diff
@@ -14,6 +14,7 @@
 
 class TestTestManifest(unittest.TestCase):
     def setUp(self):
+        self.maxDiff = None
```

Comment:
Noop?

Reply:
This shows a longer diff when tests fail, which is ok to leave IMO. Maybe there's a way to enable it globally ...